### ISSUE TITLE
[BUG] An error occured when I using the provided demo

### ISSUE THREAD

**Issue by HerbertHe** [reaviz/reaviz#7]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

When I using the provided `demo` and `data` of component `CalendarHeatmap`, `TypeError` occured and it showed that `date.getFullYear() is not a Function` and `date.getMonth() is not a Function` in `index.cjs.js`. I have checked for the reason, I guess maybe the type of parameter `date` results in this error.

## Expected behavior

Demo can be run well!

## Environment

<pre><code>
Libs:
- react version: 16.13.1
- reaviz version: 8.1.8
- Typescript version: 3.9.6

Browser:
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 12.14.1
- Platform:  Windows

Others:

</code></pre>

**Comment by admc** [comment]
This is a real bummer, copy pasted the demo and getting this error, any plans to sort this out in the near future?

**Comment by amcdnl** [comment]
@admc - Can you give me some context? I've never seen this error. Maybe its bad data?

**Comment by admc** [comment]
Of course, made an app using create-react-app, copied the example code from the primary calendar full year example, and used the data in the storybook, get the exact error outlined above. https://reaviz.io/?path=/story/charts-heatmap-calendar--month-calendar
